src/bot/commands/misc/convert.js

```diff
@@ -27,12 +27,50 @@ async function fetchLatestRates () {
  * @param {number} baseValue
  * @param {string} baseType Base unit
  * @param {string} targetType Target unit
- * @returns {strung} Success message
+ * @returns {string} Success message
  * @throws {Error} Generic error if conversion fails
  */
 function convertUnits (baseValue, baseType, targetType) {
 	const conversion = convert(baseValue).from(baseType).to(targetType);
-	return `${baseValue}${baseType} is equal to ${conversion.toFixed(2)}${targetType}`;
+	const fractionDigits = getFractionDigits(conversion);
+	return `${baseValue.toString()}${baseType} is equal to ${conversion.toFixed(fractionDigits)}${targetType}`;
+}
+
+/**
+ * Gets the count of zeroes after the decimal point but before a non-zero number
+ * @param {number} value
+ * @returns {number} Number of zeroes
+ */
+function getCountOfZeroesAfterDecimalPoint (value) {
+	let fractionalPart = value % 1;
```

Comment:
This is clever, I never knew this trick!

Reply:
Yeah, I got it from a stackoverflow answer